README.md

```diff
@@ -1,4 +1,750 @@
+			if err != nil {
+				fmt.Printf("failed to create new bearer token authenticator: %s\n", err.Error())
+				os.Exit(1)
+			}
+		}
+		// End Authenticator
+
+
+
+
+### Creating a client for the Admin REST API.
+---
+Create a new service object.
+
+		// Create Service
+		serviceAPI, serviceErr := adminrestv1.NewAdminrestV1(&adminrestv1.AdminrestV1Options{
+			URL:           URL,
+			Authenticator: authenticator,
+		})
+		// End Create Service
+
+
+
+### Creating a Kafka topic
+---
+To create a Kafka topic the admin REST SDK issues a POST request to the /admin/topics path. 
+The body of the request contains a JSON document, for example:
+{
+    "name": "topicname",
+    "partitions": 1,
+    "configs": {
+        "retentionMs": 86400000,
+        "cleanupPolicy": "delete"
+    }
+}
+
+The only required field is name is `Partitions` which is defaults to 1 if not set.
+
+Expected HTTP status codes:
+
+- 202: Topic creation request was accepted.
+- 400: Invalid request JSON.
+- 403: Not authorized to create topic.
+- 422: Semantically invalid request.
+
+If the request to create a Kafka topic succeeds then HTTP status code 202 (Accepted) is returned. If the operation fails then a HTTP status code of 422 (Un-processable Entity) is returned, and a JSON object containing additional information about the failure is returned as the body of the response.
+
+
+
+
+#### Example
+
+	func createTopic(serviceAPI *adminrestv1.AdminrestV1) error {
+		// Construct an instance of the createTopicOptionsModel.
+		createTopicOptionsModel := new(adminrestv1.CreateTopicOptions)
+		createTopicOptionsModel.Name = core.StringPtr("test-topic")
+		createTopicOptionsModel.Partitions = core.Int64Ptr(int64(26))
+		createTopicOptionsModel.PartitionCount = core.Int64Ptr(int64(1))
+	
+		// Create the Topic.
+		response, operationErr := serviceAPI.CreateTopic(createTopicOptionsModel)
+		if operationErr != nil {
+			return fmt.Errorf("Error Creating Topics: %s\n", operationErr.Error())
+		}
+	
+		// Check the result.
+		if response.StatusCode != http.StatusAccepted {
+			return fmt.Errorf("Error Creating Topic: status %d\n", response.StatusCode)
+		}
+	
+		fmt.Printf("\tname: %s created\n", *createTopicOptionsModel.Name)
+	
+		return nil
+	} // func.end
+
+
+
+
+
+
+
+
+### Deleting a Kafka topic
+---
+To delete a Kafka topic, the admin REST SDK issues a DELETE request to the `/admin/topics/TOPICNAME`
+path (where `TOPICNAME` is the name of the Kafka topic that you want to delete).
+
+Expected return codes:
+- 202: Topic deletion request was accepted.
+- 403: Not authorized to delete topic.
+- 404: Topic does not exist.
+  
+A 202 (Accepted) status code is returned if the REST API accepts the delete
+request or status code 422 (Un-processable Entity) if the delete request is
+rejected. If a delete request is rejected then the body of the HTTP response
+will contain a [JSON object](#information-returned-when-a-request-fails) which
```

Comment:
It looks like this link is broken, `information-returned-when-a-request-fails` is not the title of a section

Reply:
This all comes from Adrians orginal document https://raw.githubusercontent.com/ibm-messaging/event-streams-docs/master/admin-rest-api/README.md

Reply:
removed the link.